.github/workflows/lint.yml

```
@@ -345,6 +345,21 @@ jobs:
             --clang-tidy-exe "$(which clang-tidy)" \
             --diff-file pr.diff \
             --disable-progress-bar 2>&1 | tee "${GITHUB_WORKSPACE}"/clang-tidy-output.txt
+
+      # Run clang-tidy on a smaller subset of the codebase on master until we
+      # make the repository clang-tidy clean
+      - name: Run clang-tidy
+        if: github.event_name == 'push'
```

nit: limit to `master` since people push to other pytorch branches all the time but we don't care about those

```suggestion
        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```